cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -295,7 +295,8 @@ public void getAll(@NotNull Context ctx) {
                 if (begin != null) {
                     beginZdt = DateUtils.parseUserDate(begin, timezone);
                 } else {
-                    beginZdt = endZdt.minusHours(24);
+                    // uses earliest supported time in CWMS
+                    beginZdt = ZonedDateTime.ofInstant(Instant.ofEpochMilli(-999999999L), tz);
```

Comment:
this instant is `1969-12-20T10:13:20.001Z` which is definitely not the earliest supported time in CWMS

Reply:
Tests should be added to test edge case of the earliest value

Reply:
I realized the instant value I used is actually the minimum year allowed by CWMS.

I added a test case for the minimum time allowed by the database, which is much more recent than the CWMS allowed value.